bundle/src/main/java/com/adobe/acs/commons/httpcache/config/impl/RequestParameterHttpCacheConfigExtension.java

```diff
@@ -39,6 +39,9 @@
 import java.util.Map;
 import java.util.UUID;
 
+import static org.apache.commons.collections.CollectionUtils.containsAny;
```

Comment:
static import of methods is unclear and makes it appear these methods are implemented by this class. better to clearly qualify where they come from in code.